content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: news
+generated_by_cms: true
+title: NavCoin Core 4.3.0 Release Candidate
+author: Craig MacGregor
+date: '2018-07-11T19:11:27+12:00'
+feature_image: /images/uploads/11-07-2018.png
+categories:
+  - Nav News
+---
+This week brings your chance to test and comment on the release candidate for NavCoin Core 4.3.0. This update introduces the OpenAlias feature for NavCoin core, adds wallet support for bootstrapping, and delivers some GUI fixes.
+
+This week brings your chance to test and comment on the release candidate for NavCoin Core 4.3.0. This update introduces the OpenAlias feature for NavCoin core, adds wallet support for bootstrapping, and delivers some GUI fixes.
+
+Check out the pull request in GitHub here:
+
+<https://github.com/NAVCoin/navcoin-core/pull/232>
```

Comment:
I'd move this behind the first paragraph at least or to the very end

Reply:
I think the guys really want people to engage with the PR which is why it's at the top like that.